packages/frontend/src/components/Explorer/ExploreTree/TableTree/TableTreeSections.tsx

```diff
@@ -32,41 +32,110 @@ const TableTreeSections: FC<Props> = ({
     const sectionDepth = depth;
     const treeRootDepth = depth + 1;
     const hasNoMetrics = Object.keys(table.metrics).length <= 0;
+
+    const isSearching = !!searchQuery && searchQuery !== '';
```

Comment:
is this available from the underlying react query again?

Reply:
Oh wait just read further down and this doesn't mean "is loading search results" it means "I have filtered to only see the results from a search"